### DISCUSSION TITLE
Support for lazy loading images  <img loading=lazy>.

### DISCUSSION THREAD

**Discussion by YacineNL** [sibiraj-s/ngx-editor#328] [Ideas]
I guess it wouldn't be complicated to add since it's now supported by most browsers and those who don't simply ignore it without side effects according to web.dev.

**Comment by sibiraj-s** [comment]
Sorry for the delay. 

I am thinking of adding an image menu, that appears on selecting an image, which can have more options. This can be added to that menu. 

But for now it can also be added by creating a custom schema and a custom menu item to set the value. Let me know if you need any help regarding that.

**Comment by YacineNL** [comment]
Thank you for your response, I have found great use in your editor, I
appreciate it very much.
I  will try my best regarding that implementation, I hope I can get it to
work.

I wish you the best of luck with this project and its further development.
It works great with angular 11 and has few to no issues, I tried other
editors but they were not developer friendly at all.

Best regards,
Yacine.

**Reply by sibiraj-s** [reply]
Thanks.

Let me know if you need any help.

[a comment by an automated account is not shown]
